seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3240,7 +3242,7 @@
         "var player = testObj;   // Change this Line"
       ],
       "tail": [
-        ""
+        "if(typeof(player) !== \"undefined\"){(function(v){return v;})('\"' + player + '\"');}"
```

Comment:
Thanks @bugron!

The `typeof` is refered from below: 
`line 87 : if(typeof(myName) !== \"undefined\"){(function(v){return v;})(myName);}`

I was also wondering the need for `(` and `)` until I realized this could be a standard that perhaps FCC is following, for example `<br >`, where we are supposed to add an extra space in the closing `br` tags. You can see the same across other places.

About adding the `\"` some of the challenges show, the names as string in quotes, however I can do that change to return just the player instead of `"player"`.

But please confirm about the `typeof`.